sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/src/Internals/HttpPipelineHelper.cs

```diff
@@ -51,6 +51,23 @@ internal static int GetRetryInterval(Response httpResponse)
             return MinimumRetryInterval;
         }
 
+        internal static TimeSpan GetRetryIntervalTimespan(Response httpResponse)
+        {
+            if (httpResponse != null && httpResponse.Headers.TryGetValue(RetryAfterHeaderName, out var retryAfterValue))
+            {
+                if (int.TryParse(retryAfterValue, out var delaySeconds))
```

Comment:
If there are 10 transmissions, then the 10th item will wait for all 9 previous items to complete their execution, as we are using Interlocked.Exchange. For example, if the 10th item has a retry-after value of 10 seconds and it has already spent 10 seconds waiting on the lock, are you planning to handle this scenario?

Reply:
I am sure I understand the wait part. There is no waiting here. But, I just figured there is another issue with this.